prun.py

```diff
@@ -30,9 +30,19 @@
 logging.basicConfig(filename='paracelrun_log', format = '%(asctime)s : %(levelname)s : %(message)s', level = logging.INFO)
 logger = logging.getLogger(__name__)
 
-PARACEL_INSTALL_PREFIX='./local/'
+PARACEL_INSTALL_PREFIX = os.path.dirname(os.path.abspath(__file__))
 #PARACEL_INSTALL_PREFIX='/nfs/xxx/paracel/local'
-#PARACEL_INSTALL_PREFIX='/usr/local'
+#PARACEL_INSTALL_PREFIX='./local'
```

Comment:
Thanks for reply. What about the case users install paracel not in system folder? `LD_LIBRARY_PATH` is one way to update system linking configuration. What's your suggested way?